client/dive-common/components/RunPipelineMenu.vue

```diff
@@ -89,7 +79,8 @@ export default defineComponent({
         if (props.subTypeList.every((item) => item === 'stereo') && (name === stereoPipelineMarker)) {
           sortedPipelines[name] = category;
         } else if (props.subTypeList.every((item) => item === 'multicam') && (multiCamPipelineMarkers.includes(name))) {
-          if (name.split('-')[0] === camNumber.value.toString()) {
+          const pipelineExpectedCameraCount = name.split('-')[0];
+          if (camNumberStringArray.value.includes(pipelineExpectedCameraCount)) {
```

Comment:
This has no real world problem currently until we implement this on web. You could have a mix of multicam lengths [2,3,2,3,2] and that would allow you to run the 3-cam pipeline on all that are selected/checked?

Reply:
What is the correct behavior?  Should it show nothing?

Reply:
I think nothing unless they are all the same camera number?

Reply:
Should we solve this problem now, or switch the API to accept a single camera number instead of an array like you had it?

Reply:
I think it's fine, there is no danger currently with it.